core-feature-pack/common/src/main/resources/modules/system/layers/base/org/bouncycastle/bcpg/main/module.xml

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
+
 <!--
   ~ JBoss, Home of Professional Open Source.
-  ~ Copyright 2011, Red Hat, Inc., and individual contributors
+  ~ Copyright 2010, Red Hat, Inc., and individual contributors
```

Comment:
Why did you change the copyright year? If that's for a new file, should it be `2020`?